src/Components/Analyzers/test/TestFiles/ComponentInternalUsageDiagnoticsAnalyzerTest/UsesRenderTreeFrameAsParameter.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Components.RenderTree;
+
+namespace Microsoft.AspNetCore.Components.Analyzers.Tests.TestFiles.ComponentInternalUsageDiagnoticsAnalyzerTest
+{
+    class UsesRenderTreeFrameAsParameter
+    {
+        private void Test(/*MM*/RenderTreeFrame frame)
```

Comment:
What's `/*MM*/`? Is this just adding a comment node to show the analyzer can handle it?

Reply:
This is a marker so the test framework can compare against the location of the diagnostic.